build.sh

```diff
@@ -37,18 +35,9 @@ else
     cp "$PUBLIC_API_SWAGGER_SRC" swagger/
 fi
 
-# -- Build docs --
-echo -e "${white} Converting swagger file to shins compatible markdown...${NC}"
 cp source/images/govuk_pay_logo.png shins/source/images/logo.png
-npm install
-./node_modules/widdershins/widdershins.js swagger/swagger.json -o shins/source/index.html.md
-
-echo -e "${white} Building static docs...${NC}"
-cd shins
-npm install
-node shins.js --minify
-
-# -- Move static files to 'build' folder --
-mkdir -p build
-cp -R index.html pub source build/
-echo -e "${green}Built pay-api-docs static files...${NC}"
+
+# -- Build docs inside docker container --
+docker build --rm -t govukpay/pay-api-docs-generator:local .
```

Comment:
This is rather racy - if we run two jobs at the same time you can't guarantee what image the following `docker run` picks up

Reply:
I don't think you're going to make this non-racy very easily so I'm OK with this as is

Reply:
Updated to use image ID generated by docker instead. Image should now always be unique to the job